packages/react/src/Switch/Switch.test.tsx

```diff
@@ -0,0 +1,106 @@
+import { render, screen } from '@testing-library/react'
+import { createRef } from 'react'
+import { Switch } from './Switch'
+import '@testing-library/jest-dom'
+
+describe('Switch', () => {
+  it('renders an element with role switch', () => {
+    render(<Switch />)
+    const switchElement = screen.getByRole('switch')
+
+    expect(switchElement).toBeInTheDocument()
+  })
+
+  it('renders the Switch with the default state unchecked', () => {
```

Comment:
```suggestion
  it('is not checked by default', () => {
```

All these tests are already grouped in the 'Switch' `describe`, so it's not necessary to repeat that here imo